.jenkins/rocm/build_and_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# exit immediately on failure, or if an undefined variable is used
+set -eux
+
+FBGEMM_REPO_DIR=${1:-/workspace/FBGEMM}
+
+git config --global --add safe.directory $FBGEMM_REPO_DIR
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/asmjit
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/cpuinfo
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/googletest
+git config --global --add safe.directory $FBGEMM_REPO_DIR/third_party/hipify_torch
+
+# Install dependencies
+apt-get update --allow-insecure-repositories && \
+  apt-get install -y --allow-unauthenticated \
+  git \
+  jq \
+  sshfs \
+  sshpass \
+  unzip
+
+apt-get install -y locales
+locale-gen en_US.UTF-8
+
+pip3 install click
+pip3 install jinja2
+pip3 install ninja
+pip3 install scikit-build
+pip3 install --upgrade hypothesis
+pip3 install --pre torch torchvision --extra-index-url https://download.pytorch.org/whl/nightly/rocm5.1.1/
+
+pip3 list
+
+# Build fbgemm_gpu
+cd $FBGEMM_REPO_DIR/fbgemm_gpu
+export MAX_JOBS=`nproc`
```

Comment:
**[(style) Use $(...) notation instead of legacy backticks ....]**
Can I ask you to use "$(nproc)" instead of `nproc`?
**[(warning) Declare and assign separately to avoid masking return values.]**
This error: https://github.com/koalaman/shellcheck/wiki/SC2155

So can we rewrite it to the following?
```sh
MAX_JOBS="$(nproc)"
export MAX_JOBS
```

Reply:
Thank you for pointing it out. It was rewritten.